blockly/src/entities/BlocklyMonster.java

```diff
@@ -0,0 +1,177 @@
+package entities;
+
+import com.badlogic.gdx.audio.Sound;
+import components.TintViewDirectionComponent;
+import contrib.components.AIComponent;
+import contrib.entities.AIFactory;
+import contrib.entities.MonsterDeathSound;
+import contrib.entities.MonsterFactory;
+import contrib.entities.MonsterIdleSound;
+import contrib.utils.EntityUtils;
+import contrib.utils.components.skill.Skill;
+import core.Entity;
+import core.components.PositionComponent;
+import core.utils.Point;
+import core.utils.components.MissingComponentException;
+import core.utils.components.path.IPath;
+import core.utils.components.path.SimpleIPath;
+import java.io.IOException;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import utils.components.ai.fight.StraightRangeAI;
+import utils.components.skill.InevitableFireballSkill;
+
+/**
+ * Enum representing the different types of monsters in Blockly.
+ *
+ * <p>Each type of monster has:
+ *
+ * <ul>
+ *   <li>A name
+ *   <li>A texture path
+ *   <li>A health value
+ *   <li>A speed value
+ *   <li>A chance to drop an item
+ *   <li>A death sound
+ *   <li>A fight AI
+ *   <li>An idle AI
+ *   <li>A transition AI
+ *   <li>A collide damage value
+ *   <li>A collide cooldown value
+ *   <li>An idle sound path
+ * </ul>
+ *
+ * <p>Each monster type can be built into an entity using the {@link #buildMonster(Point,
+ * PositionComponent.Direction)} method.
+ */
+public enum BlocklyMonster {
+  /** A static non-moving guard monster. */
+  GUARD(
+      "Blockly Guard",
+      "character/monster/big_daemon",
+      99999, // immortal
```

Comment:
```suggestion
      1,
```

die müssen nicht unsterblich sein. Der default gegener geht mit einen FB kaputt.